docs/manuals/source/Appendix/HardwareSizing.rst

```diff
@@ -36,31 +39,41 @@ Depending on the number of files and the average length of filenames, the **data
 
 To **calculate the number of files** in the DB, the number of files being backed up from all systems needs to be multiplied by the number of times that they will be kept in the database.
 
-The **amount of data per file** in the DB, depends on the size of filenames that are being backed up, but we have analyzed some real-world examples and found that values between **250 and 350 bytes per row** are usual.
+The **amount of data per file or path** in the DB, depends on the size of filenames or path length that are being backed up, but we have analyzed some real-world examples and found that values between **250 and 350 bytes per row** are usual.
 
-So the calculation of the size of the file table can be approximated with the following formula:
+The size of the path indexes is around 60% of the table size.
+
+So the calculation of the size of the file and path tables can be approximated with the following formula:
 
 .. math::
    \begin{split}
-   s &= n_f \cdot n_b \cdot 300 \frac{\mbox{bytes}}{\mbox{row}} \\
+   s &= n_f \times n_b \times 300 \frac{\mbox{bytes}}{\mbox{row}} \\
+   p &= \frac{s}{10} \\
    s &: \mbox{storage required for file table} \\
    n_f &: \mbox{number of files in a (full) backup} \\
    n_b &: \mbox{number of (full) backups} \\
+   p &: \mbox{storage required for path table} \\
    \end{split}
 
-*Example:* If **200.000 files** are backed up during a full backup, a full backup is run **every week** and the retention of the backups is **4 weeks**, the total amount of files would be
+*Example:* If **2.000.000 files** are backed up during a full backup, a full backup is run **every week** and the retention of the backups is **4 weeks**, the total amount of files would be
 
 .. math::
    \begin{split}
-   n_f &= 200.000\ \mbox{Files} \\
+   n_f &= 2.000.000\ \mbox{Files} \\
```

Comment:
Have you checked that these numbers are true for current databases? (Number of path, file entries compared to total db size in real installations)?

Reply:
Here, we propose some mathematical way to approximate the size of the database. Again the formula can be checked against the real world example we got from support and bug entries.